build_tools/scripts/lint.sh

```diff
@@ -139,6 +139,13 @@ echo "***** Generates CMake files *****"
 git add -A
 git diff HEAD --exit-code
 
+echo "***** Check BUILD files are not named BUILD (prefer BUILD.bazel) *****"
+if [[ $(git ls-files '**BUILD') ]]; then
```

Comment:
Should probably be consistent about `**BUILD` vs `**/BUILD`.

Reply:
Thanks for the catch.